addons/pharma/stringtable.xml

```diff
@@ -3829,6 +3898,7 @@
             <Dutch>Vereist medisch niveau voor gebruik Coag-Sense</Dutch>
             <French>Niveau médical requis du Coag-Sense</French>
             <Italian>Livello medico richiesto per Coag-Sense</Italian>
+            <German>Für Coag-Sense ist ein medizinisches Niveau erforderlich</German>
```

Comment:
This sentence says that a medical level is required for the coag sense, it should instead say which medical level is required.
"Erforderliches medizinisches Niveau für den Coag-Sense"